index.html

```diff
@@ -915,37 +1057,117 @@ <h4>Text</h4>
-            language (original), or if it is a representation in another language (translation):</p>
-          <ul>
-            <li><dfn>Original</dfn> - the <a>Text</a> is any of:
-              <ol>
-                <li>the same language as the dialogue that it represents in the original programme audio;</li>
-                <li>a transcription of text visible in the original programme video, in the same language as that text;</li>
-                <li>representative of non-dialogue sound and is in the <a>Primary Language</a> of the <a>DAPT Script</a>;</li>
-                <li>representative of the scene in the original programme video, and is in the <a>Primary Language</a> of the <a>DAPT Script</a>.</li>
-              </ol>
-            </li>
-            <li><dfn>Translation</dfn> - the <a>Text</a> is a representation of the Original language text in a different language.</li>
-          </ul>
-          <aside class="note">The language of a <a>Text</a> object whose <a>Text Language Source</a> is <a>Original</a>
-            can be different to the document's <a>Primary Language</a>.
-            For example a <a>transcript</a> of a short piece of dialogue spoken in a different language to the majority of the dialogue would be annotated
-            with both a different <code>xml:lang</code> value and as being <a>Original</a>.</aside>
-          <p>The <a>Text Language Source</a> property is represented in a <a>DAPT Document</a> by a <code>daptm:langSrc</code> attribute with the following constraints:</p>
-          <p class="ednote">Should we use an abbreviated attribute name?</p>
-          <p class="ednote">Initial design is to use an abbreviated name and <code>original|translation</code>,
-            though I considered using an abbreviated value too, since this attribute will appear on every <code>&lt;p&gt;</code> element.
-            Abbreviating O for Original is probably a bad idea because the letter O and the number 0 can easily be confused.
-            I also considered P for Primary but that caused potential confusion between Primary Language and Primary Text Language Source.</p>
+          <p>The <dfn>Text Language Source</dfn> property is an annotation indicating the source language of
+            a <a>Text</a> object, if applicable, or that the source content had no inherent language.</p>
+          <p>If it is not empty and differs from the language of the <a>Text</a> object,
+            that implies that it is a representation in another language,
+            referred to as &quot;translation&quot;;
+            conversely if it is the same that implies that the language of the <a>Text</a> is <a>Original</a>.</p>
+          <p>An empty <a>Text Language Source</a> property on a <a>Text</a> object
+            indicates that the <a>Text</a> represents content without an inherent language,
+            such as untranslated descriptions of a visual scene or
+            captions representing non-dialogue sounds.</p>
```

Comment:
I'm happy to make this more structured. The proposal you make here mixes up whether the value is specified or not and the interpretation of the computed value, so it will need some adjustment. There is an if/else for working out the computed value based on what is specified, and then there's an if/else for working out if it's Original or Translation based on that computed value.

Reply:
I'm not sure what you mean by "mixes up", I just added if/else and bullet points, the text was already there.

Reply:
Upon reading your proposed text, I'm fine with it. 

One minor suggestion: I would move the if/Otherwise bullets before the sentence "Text Language Source is an inheritable property.". The If/Otherwise part goes nicely with the first sentence of the section, which I would also end with `:` to emphasize that the bulleted list is providing more precise definition.